assessments/intake-process.md

```diff
@@ -50,12 +50,19 @@ assessment, exceeding the bandwidth of the group:
    incubated projects, then sandbox).
 4. Non-CNCF Projects that request a review (or invited by SIG members).
 
-Security Assessment Facilitator, in collaboration with the named chair,
-has the discretion to adjust priority in order to streamline the process,
-or per their own judgement for other reasons consistent with SIG-Security
-mission and charter.  In all cases, the priority queue will be maintained
-transparently as described below, along with communication via regular
-chair-liaison meetings and SIG-Security reports at TOC meetings.
+The Security Assessment Facilitator, in collaboration with the named chair, has
+the discretion to adjust priority in order to streamline the process, or per
+their own judgement for other reasons consistent with SIG-Security mission and
+charter.  In all cases, the priority queue will be maintained transparently as
+described below, along with communication via regular chair-liaison meetings and
+SIG-Security reports at TOC meetings.
+
+The Security Assessment Facilitator with concurrence from one (1) co-chair will
+determine if a CNCF or non-CNCF project may be accepted into the assessment
+queue.  If at any time, the project's requesting review cease communicating, the
+Security Assessment Facilitator may remove the project from the queue with
```

Comment:
I guess that marking the issue is a way to do this?  I just tried the process on Falco, but really no one got a chance to weigh in.  I don't know if this is what we want or not.

Reply:
I think we want to give you enough leeway that if you find it is stalled with no responses to inquiries, you should be empowered to manage the queue accordingly, which may mean kicking them from the queue with public posting as to why ( a la issue update).  Definitely marking the issue.  
If you are looking for something more process specific with timelines how does the following look?
* If after 3 communication attempts by the Security Assessment Facilitator, Lead Reviewer, or a co-chair and the project ceases to respond the Security Assessment Facilitator may comment on the assessment issue and if no further updates are posted within 7 days, they will close the issue with notification to the co-chairs.

Reply:
I'd rather not be so prescriptive about the communication / timeline.  (If we were going to automate it, this is fine, but I won't remember these exact rules.)

I was more trying to point out that there wasn't really a chance for Falco to respond.  That may be fine, but we should be aware...